js/pages/configuration/users-import/browser.js

```diff
@@ -0,0 +1,104 @@
+define([
+		'knockout',
+		'appConfig',
+		'text!./browser.html',
+		'services/AuthAPI',
+		'utils/AutoBind',
+		'components/Component',
+		'utils/CommonUtils',
+		'utils/DatatableUtils',
+		'./services/JobService',
+		'./const',
+		'./utils',
+		'less!./browser.less',
+], function(
+		ko,
+		config,
+		view,
+		authApi,
+		AutoBind,
+		Component,
+		commonUtils,
+		datatableUtils,
+		jobService,
+		Const,
+		Utils,
+) {
+		class UserImportBrowser extends AutoBind(Component) {
+
+			constructor(params) {
+				super();
+				this.config = config;
+				this.isAuthenticated = authApi.isAuthenticated;
+				this.canImport = ko.pureComputed(() => this.isAuthenticated() && authApi.isPermittedImportUsers());
```

Comment:
`isAuthenticated()` is redundant - there is such check inside `AuthAPI.isPermitted` used by `isPermittedImportUsers`